.circleci/config.yml

```diff
@@ -77,9 +77,24 @@ workflows:
           tf_tier: shared
           tf_workspace: development
 
+      - apply:
```

Comment:
I feel like the naming for this job should change now - we're using apply for plan, apply and destroy. Struggling to think of a catchall though that snappy. Something like `run terraform command` is descriptive. I'd also drop the default for apply if you decide to change the job name and be explicit about passing it in when we want to run apply.

Reply:
Yeah i did want to do that. I call it terraform action in my other integration pipelines but didn't want to make too many changes and risk borking the pipeline but you're right, let's go for it. Might as well do it now.